easybuild/easyconfigs/y/YODA/YODA-1.9.7-gompi-2022a.eb

```
@@ -0,0 +1,40 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Alexander Puck Neuwirth <alexander@neuwirth-informatik.de>
+
+easyblock = 'ConfigureMake'
+
+name = 'YODA'
+version = '1.9.7'
+
+homepage = 'https://yoda.hepforge.org/'
+description = """
+Yet more Objects for (High Energy Physics) Data Analysis
+"""
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+source_urls = ['https://yoda.hepforge.org/downloads?f=']
+sources = ['%(name)s-%(version)s.tar.bz2']
+checksums = ['8d07bb04dcb79364858718a18203452d8d9fa00029fa94239eafa8529032b8ff']
+
+builddependencies = [
+    ('Autotools', '20220317'),
+]
+
+dependencies = [
+    ('Python', '3.10.4')
+]
+
+preconfigopts = "sed -i 's|^\  YODA_PYTHONPATH=.*|\  "
+preconfigopts += "YODA_PYTHONPATH=%(installdir)s/lib/python%(pyshortver)s/site-packages|g' configure &&"
+configopts = "--disable-root --enable-pyext --disable-static"
```

we can actually avoid `sed`ing this, since we can override the `YODA_PYTHONPATH` variable during the `make` and `make install` steps:

```suggestion
configopts = "--disable-root --enable-pyext --disable-static"
buildopts = "YODA_PYTHONPATH=%(installdir)s/lib/python%(pyshortver)s/site-packages"
installopts = buildopts
```